jarvis_files/jarvis_cmd/launchScripts/dim

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+dimensions=$(xdpyinfo | grep dimensions)
```

Comment:
will people have to install xdpyinfo and xdotool using apt?

Reply:
Yes